rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -122,13 +124,26 @@ def __init__(
         if not HAS_NOTEBOOK:
             raise RerunMissingDependencyError("rerun-notebook", "notebook")
         self._error_widget = _ErrorWidget()
+
+        # Get access token from env variable
+        fallback_token = os.environ.get("REDAP_TOKEN", None)
```

Comment:
What is this for? It seems redundant with the `set_credentials` below. I think the `fallback_token` should be passed through as-is like before, and credentials set if they exist, in parallel